lib/runner.js

```diff
@@ -400,7 +400,8 @@ class Runner extends Emittery {
 					err: serializeError('Test failure', true, result.error, this.file),
 					duration: result.duration,
 					knownFailing: result.metadata.failing,
-					logs: result.logs
+					logs: result.logs,
+					testFile: this.file
```

Comment:
Why is this needed? IIRC the code that handles these changes knows what file they come from.